docs/build_process.rst

```diff
@@ -3,6 +3,148 @@
 Understanding the Build Process
 ===============================
 
+General workflow
+----------------
+
+.. image:: images/osbs2_detail.png
+   :alt: OSBS workflow diagram
+
+
+OSBS build starts either with fedpkg or koji command. It creates buildContainer/buildSourceContainer
+task in koji hub. When osbs brew builders in channel container-binary/container-source have
```

Comment:
Isn't "container-binary" and "container-source" a detail of the internal deployment? Do we want this upstream?

Reply:
ok removed